core/src/components/input-password-toggle/input-password-toggle.ionic.scss

```
@@ -0,0 +1,12 @@
+@use "../../foundations/ionic.vars.scss" as tokens;
+@import "./input-password-toggle.common.scss";
+
+// Ionic Input Toggle Password
+// --------------------------------------------------
+:host ion-button.button-has-icon-only {
+  --padding-end: inherit;
+  --color: #{tokens.$ionic-color-neutral-500};
+  --margin-right: calc(-1 * var(--padding-end));
+
+  margin: 0 var(--margin-right) 0 0;
```

```suggestion
  --margin-end: calc(-1 * var(--padding-end));

  @include margin(0, var(--margin-end), 0, 0);
```

We should use `margin-end` to match RTL naming